test/modules/TestResultCoordinator/IReportMetadata.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator
+{
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    interface IReportMetadata
+    {
+        string ExpectedSource { get; }
+
+        string ActualSource { get; }
+
+        ReportType ReportType { get; }
+
+        TestOperationResultType TestOperationResultType { get; }
+
+        public string ToString();
```

Comment:
Don't need to force to implement ToString().  if not implemented, it just can't print nicely for debug.